internal/pkg/api/schema.go

```diff
@@ -175,3 +175,47 @@ type StatusResponse struct {
 	Status  string                 `json:"status"`
 	Version *StatusResponseVersion `json:"version,omitempty"`
 }
+
+type FileInfo struct {
+	ActionID string     `json:"action_id"`
+	AgentID  string     `json:"agent_id"`
+	Source   string     `json:"src"`
+	File     FileData   `json:"file"`
+	Contents []FileData `json:"contents"`
+	Event    struct {
+		ID string `json:"id"`
+	} `json:"event"`
+	Host struct {
+		Hostname string `json:"hostname"`
+	} `json:"host"`
+}
+
+type FileData struct {
+	Size        int64  `json:"size"`
+	Name        string `json:"name"`
+	Extension   string `json:"ext"`
+	Mime        string `json:"mime_type"`
+	Compression string `json:"Compression"`
+	Hash        struct {
+		SHA256 string `json:"sha256"`
+		MD5    string `json:"md5"`
+	}
```

Comment:
(nit) Here Hash will be capitalized as well

Reply:
good catch, this one is ecs and should be lowercase